sdl/_testdata/simple.yaml

```diff
@@ -1,22 +1,22 @@
 ---
-version: "1.5"
-
+version: "2.0"
 services:
   web:
     image: nginx
     expose:
       - port: 80
         to:
           - global: true
-
 profiles:
-
   compute:
     web:
-      cpu: "100m"
-      memory: "128Mi"
-      storage: "1Gi"
-
+      resources:
+        cpu:
+          units: "100m"
+        memory:
+          amount: "128Mi"
+        storage:
+          amount: "1Gi"
```

Comment:
`amount` -> `size`?

Reply:
previously it was `Size` member in Storage and Memory, but protobuf generates own `Size()`. can have field named `Amount` or `Sz` but tags as `size`?

Reply:
Can we keep it `size` in the yaml?

Reply:
yes